src/styles/DisplayComponent.css

```diff
@@ -1,17 +1,31 @@
-.displayButtons {
-  position: relative;
-  left: 40%;
-  padding: 10px;
-  margin: 16px;
-  border-radius: 15px;
-  cursor: pointer;
+.ReactVirtualized__Table__Grid {
```

Comment:
Is this a recommended way to style `react-virtualized`? I'm worry if it's not standard, the styles might not apply anymore once new `react-virtualized` version is installed and their developers decided to change classes' names.